learning-team/2020-06/2020-06-04.md

```diff
@@ -0,0 +1,82 @@
+
+## Topics
+
+If you would like to add a topic to the agenda please add a suggestion to the PR using the following format:
+
+```markdown
+### Topic goes here (your initials - 00mins)
+
+description of topic goes here
+```
+
+### replacing `ember-ref-modifier` (@chriskrycho  - CK)
+
+One of the advanced examples in the guides, [Communicating Between Elements in
+a Component][example], uses `ember-ref-modifier` to show how to implement some
+custom functionality. This example dates to a time before the existence of
+`ember-modifier`, and was reasonable to include at the time. However, today the
+example can be implemented much more cleanly using a class-based modifier from
+`ember-modifier`. Just as importantly, using `ref` has significant performance
+implications that are *not* obvious: it requires two rendering passes. I’ve had
+to wave people off of using `ref` a couple times already in our app as we adopt
+Octane, because it’s officially recommended in the Guides, which we point people
+to as our *first* source of truth for how to write Octane. I suggest we remove
+it in favor of another example using one of the `ember-modifier` variants.
+
+[example]: https://guides.emberjs.com/release/components/template-lifecycle-dom-and-modifiers/#toc_communicating-between-elements-in-a-component
+- Echoed by @rwjblue
+
+- Alternate suggestion by @jrjohnson, CK has comments on this will mention at the meeting: https://github.com/emberjs/core-notes/pull/306#issuecomment-632801292
+- Aside from @lifeart: https://github.com/emberjs/core-notes/pull/306#issuecomment-633190688
+
+### Ember CLI API docs (from @mehulkar, who may or may not attend - 10mins)
+
+I want to get an update on getting API docs on `https://api.emberjs.com/ember-cli`
+and redirecting `ember-cli.com/api`. As embroider approaches and the larger
+JS community experiments with JS development (e.g. Snowpack), it is
+important for the Ember project to _enable_ its community to experiment also.
+Moving the API docs to the main site will make it easier to improve
+documentation, add examples, and propose API improvements that can
+enable such experimentation.
+
+### Future plans for Ember Octane vs. Classic Cheat Sheet (@ijlee2, 5-10 min)
+
+The project is now an Octane app. What further developments would we like to see happen?
+
+I would like to work on updating CI (upgrade workflow actions to `v2` + introduce Percy) and look into if we need more tests to help contributors feel confident in making changes.
+
+Luke Melia provided suggestions for additional topics in two issues: ([#36](https://github.com/ember-learn/ember-octane-vs-classic-cheat-sheet/issues/36) and [#37](https://github.com/ember-learn/ember-octane-vs-classic-cheat-sheet/issues/37)). Would these topics be a good addition to the cheat sheet?
+<!-- replace with topic -->
```

Comment:
```suggestion

### Titles  in "Octane Upgrade Guide" (RM, 10min)
- https://guides.emberjs.com/release/upgrading/current-edition/
>  The titles are named after the target API instead of the thing you want to upgrade from, which makes finding relevant information tricky.

```